backend/infrahub/computed_attribute/tasks.py

```diff
@@ -159,10 +163,10 @@ async def trigger_update_python_computed_attributes(
 
 
 @flow(
-    name="process_computed_attribute_value_jinja2",
-    flow_run_name="Update value for computed attribute {attribute_name}",
+    name="computed_attribute_jinja2_update_value",
```

Comment:
If we are changing the name should we also swap out the underscores to dashes as per the discussion last week?